docs/faq.md

```diff
@@ -25,3 +25,84 @@ your campaign, depending on your settings for the
 {attr}`~baybe.campaign.Campaign.allow_recommending_already_recommended` flags.
 ```
 
+## Checklist for Designing BayBE Optimization Campaigns
+
+This checklist collects **common questions that you need to ask yourself** when designing a BayBE optimization campaign. It also provides documentation references that will help you with the corresponding setup. 
+
+Note that this is not a comprehensive guide of all BayBE's functionalities, but rather a "quick start" meant to help you with the most basic design principles.
+
+### Defining Targets
+
+```{dropdown} Should the target value be maximized, minimized, or be matched to a specific value?
+
+Specify this when
+[defining the target](userguide/targets.html#numericaltarget).
+```
+
+```{dropdown} Should multiple target be optimized simultaneously?
+
+See how to use 
+[multi-target objectives](userguide/objectives.html).
+```
+
+### Defining Parameter Search Space
+
+```{dropdown} Are only some parameter values of interest/possible?
+
+See how to exclude some 
+[parameter values](userguide/getting_recommendations.html#excluding-configurations) 
+from being recommended, such as by defining
+[bounds for continuous parameters](userguide/parameters.html#numericalcontinuousparameter)
+or [active values for discrete parameters](userguide/parameters.html#discrete-parameters).
+```
+
+```{dropdown} Are only some parameter combinations of interest/possible?
+
+See how to exclude some 
+parameter combinations from being considered by using 
+[constraints](userguide/constraints.html) or
+[constrained searchspaces](userguide/searchspace.html#creating-a-simplex-bound-discrete-subspace).
+Alternatively, if the aim is to use only a few specific parameter configurations the search space can be created from a 
+[dataframe](userguide/searchspace.html#id3) 
+rather than from the product of all possible parameter combinations.
+```
+
+```{dropdown} Are some parameters non-numeric or allow only discrete numbers?
+
+  > Use [discrete](userguide/parameters.html#discrete-parameters)
```

Comment:
#consistency 

Why do we sometimes have the leading `>`, which gives a very different rendering style?